### DISCUSSION TITLE
Ritsa TV

### DISCUSSION THREAD

**Discussion by peaking2** [iptv-org/community#764] [Q&A]
### Channel Name

Ritsa

### Channel ID

RitsaTV.ru

### Website

https://ritsatv.ru

### Notes

Music video channel

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
https://www.twitch.tv/ritsatv

**Comment by BellezaEmporium** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/ritsatv

**Comment by peaking2** [comment]
Thank you for the assistance. I'm hoping it can be added to the IPTV list. I'm still learning about the GitHub project and input.

**Comment by peaking2** [comment]
Is it possible to convert the stream or channel to m3u or m3u8. I had input the link to a playlist but it doesn't work. I read that it might be different with Twitch streams though. It has been a long term stream for a while now. And some sister channels like Eurodance 90 are in the IPTV collection.

**Comment by freearhey** [comment]
https://www.twitch.tv/ritsatv
<img width="318" alt="image" src="https://user-images.githubusercontent.com/7253922/186732457-3f3acc01-0b7c-4474-94fd-5ed6357e6d75.png">

On the website at the same time the broadcast is working:
<img width="522" alt="image" src="https://user-images.githubusercontent.com/7253922/186732860-937483cf-3d30-4cab-9e78-d8b364d9a51f.png">
https://ritsatv.ru/movie/id900777-ritsa-tv

**Comment by Alstruit** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://vk.com/video-122792749_456241869

**Comment by freearhey** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://vk.com/video-122792749_456241869 (offline)
<img width="645" alt="image" src="https://user-images.githubusercontent.com/7253922/232225325-48ffff8b-816e-4ec0-be11-c8e2054981d2.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19763
